hips/tiles/surveys.py

```diff
@@ -150,6 +150,20 @@ def tile_access_url(self, order: int, ipix: int) -> str:
         """
         return self.base_url + '/Norder' + str(order) + '/Dir' + str(self.directory(ipix)) + '/'
 
+    def tile_path(self, order: int, ipix: int, tile_format: str) -> str:
+        """Tile access URL
+
+        Parameters
+        ----------
+        order : int
+            HiPS order
+        ipix : int
+            Index of the HiPS tile
+        tile_format : str
+            HiPS tile URL
```

Comment:
Parameter description isn't correct for `tile_format`.

Also: maybe add an `Examples` section with one example?